root/instrument/List.js

```diff
@@ -15,59 +16,62 @@ const {l} = require('../static/scripts/common/i18n');
 const {lp_attributes} = require('../static/scripts/common/i18n/attributes');
 const {l_instrument_descriptions} = require('../static/scripts/common/i18n/instrument_descriptions');
 
+type PropsT = {
+  instrument_types: $ReadOnlyArray<InstrumentTypeT>,
+  instruments_by_type: {|
+    +[number]: $ReadOnlyArray<InstrumentT>,
+    unknown: $ReadOnlyArray<InstrumentT>,
```

Comment:
This is now missing the `+` on all the props expect `[number]` (did you say some eslint rule was removing those? we should disable it)

Making `PropsT` an exact object type would be nice too: `{| ... |}`